doi/src/main/java/ca/nrc/cadc/doi/VospaceDoiClient.java

```diff
@@ -178,19 +185,26 @@ public Resource getResource(String doiSuffixString, String doiFilename)
     public boolean isCallerAllowed(Node node, Subject adminSubject) {
```

Comment:
You didn't create this method, but you you rename it to something more descriptive. Is CallerAllowed to do what? Is the caller allowed to read the DOI, or write the DOI? Maybe hasReadAccess() or hasWriteAccess() or anything else that describes what the method does.